doc/articles/uwp-vs-winui3.md

```diff
@@ -35,11 +35,11 @@ The questions below should help guide you to the best choice when creating a new
 
 #### Do you use 3rd-party libraries?
 
-Because WinUI 3 is still in preview, compatible versions of 3rd-party libraries are not yet available in many cases. If you depend (or expect to depend) on these libraries, the UWP XAML API may be the way to go for now.
+Evaluate 3rd party offerings from 3rd party vendors for UWP and WinUI. While 3rd party vendors have offerings for both, UWP has been in the market longer. From Uno PLatform perspective either will work, or if not, contact us at our Discord channel.
```

Comment:
```suggestion
While 3rd party vendors have offerings for both UWP and WinUI, UWP has been in the market longer. From Uno Platform perspective either will work, or if not, contact us at our [Discord channel](https://discord.com/invite/eBHZSKG).
```

I'm not sure I understand the use of this section though, since we are moving make WinUI the default target, should we be leaning more towards pointing people to 3rd party offerings for WinUI?

Reply:
For now I wanted to edit existing content. You raise a good point, we may need do add another section to address the WinUI/WCT components too.